### ISSUE TITLE
Rename Listing Features in Options to Listing Details

### ISSUE THREAD

**Issue by JoeHana** [wpsight/wpcasa#37]
In Options people can change the standard details by checking "Listing Features". This can be a little confusing since Listing Features are actually more like tags which are managed through a custom taxonomy. Also the "Standard Features" should rather get called "Standard Detail" or maybe even better just "Detail" with a hint in the description that the default details can be set here.

**Comment by fabianmarz** [comment]
Yes, the "Listing Features" meant here are more like the fundamental object informations. "Details/Property Details/Listing Details" seems more accurate to me, too.
